vignettes/GMAT-GMST.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: "Example: Calculating GMST from GMAT in Hector"
+author: "Leeya Pressburger"
+date: "2021-08-18"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteEncoding{UTF-8}
+  %\VignetteIndexEntry{Example: Carbon tracking with Hector}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Introduction
+
+The global mean surface temperature (GMST) is the average of the air temperature over land and the temperature at the surface of the ocean. This is a very useful metric for observing global warming or global cooling. However, Hector does not produce GMST as an output at this time. Hector does measure the global mean air temperature (GMAT), which we can use to back-calculate GMST. 
+
+This vignette will demonstrate how to set up the calculation between GMAT and GMST then plot results to observe differences over time. 
+
+## Setup
+
+Let's load the packages we will need and run a scenario.
+
+*** will have to recode without dplyr/tidyr - might need help with this
+
+``` {r hector.init}
+library(hector)
+library(ggplot2)
+library(dplyr)
+library(tidyr)
+
+inifile <- file.path(system.file('input', package = 'hector'),
+                     'hector_rcp45.ini')
+core <- newcore(inifile)
+run(core)
+```
+
+
+## Find GMST
+
+In order to find the GMST from Hector outputs, we will need to calculate it from the ocean surface temperature and the land air temperature. This can be done by finding a weighted sum of the land and ocean temperature. 
```

Comment:
Consider simplifying: "GMST can be calculated from Hector outputs by computing a weighted sum of the land air and ocean surface temperatures."